modules/plugin/jdbc/jdbc-mysql/src/main/java/org/geotools/data/mysql/MySQLDataStoreFactory.java

```diff
@@ -92,14 +101,58 @@ protected JDBCDataStore createDataStoreInternal(JDBCDataStore dataStore, Map par
         if (storageEngine == null) {
             storageEngine = (String) STORAGE_ENGINE.sample;
         }
+
+        Boolean enhancedSpatialFlag = (Boolean) ENHANCED_SPATIAL_SUPPORT.lookUp( params );
+        if (enhancedSpatialFlag == null) {
+        	//enhanced spatial support should be enabled if MySQL 
+        	//version is at least 5.6.
+        	enhancedSpatialSupport = isMySqlVersion56(dataStore);
+        } else if (enhancedSpatialFlag && !isMySqlVersion56(dataStore)) {
+        		System.out.println("MySQL version does not support enhancedSpatialSupport. Disabling it");
```

Comment:
Eeek, a system out! 
Also, don't use tabs, indentation is done only using spaces.
Was it here already? Did not see it before sorry!

Reply:
Hi @aaime,
I was not aware that it was such a hard constraint to avoid system out since there are quite a lot of them in other modules! Anyway check the latest commit where I have removed it based on your requested change.
By the way, I have and a couple of System.out in the test packages. Is it a requirement to also remove them?

Reply:
@nprigour the code base is 15+ years old and there are indeed many bad things in there from a time when "anything went" (system outs, un-closed resources, free formatting for all, eaten exceptions, you name it... wish we could do a "clean up" code sprint, but we systematically fail to sponsor one... we try to clean up things a bit while we edit a certain code area for other reasons).
It test packages normally system outs are commented out prior to commit, or better, replaced with logging.